pkg/registry/certificates/certificates/storage/storage.go

```diff
@@ -62,6 +62,7 @@ func NewREST(optsGetter generic.RESTOptionsGetter) (*REST, *StatusREST, *Approva
 	statusStore := *store
 	statusStore.UpdateStrategy = csrregistry.StatusStrategy
 	statusStore.ResetFieldsStrategy = csrregistry.StatusStrategy
+	statusStore.BeginUpdate = countCSRDurationMetric(csrDurationRequested, csrDurationHonored)
```

Comment:
hurggggggg

Reply:
🤣 open to alternatives

Reply:
the characteristics you wanted were "access to old and new object", and "guaranteed to run at most once per persisted status update that issues the cert?"

I don't see another place for both of those :-/